CMakeLists.txt

```diff
@@ -0,0 +1,79 @@
+cmake_minimum_required(VERSION 3.4)
+project(speexdsp)
+
+include(CheckTypeSize)
+check_type_size("int16_t" SIZEOF_INT16)
+
+set(SPEEXDSP_FOLDER ${CMAKE_CURRENT_SOURCE_DIR})
+set(SPEEXDSP_SOURCE ${CMAKE_CURRENT_SOURCE_DIR}/libspeexdsp)
+set(SPEEXDSP_INCLUDE ${CMAKE_CURRENT_SOURCE_DIR}/include)
+
+set(SIZE16 "int16_t")
+set(USIZE16 "uint16_t")
+set(SIZE32 "int32_t")
+set(USIZE32 "uint32_t")
+
+set(HAVE_STDINT_H 1)
+set(VAR_ARRAYS 0)
+set(USE_ALLOCA 0)
+set(EXPORT "__attribute__((visibility(\"default\")))")
+set(ENABLE_VALGRIND 0)
+set(FIXED_POINT 0)
+set(FLOATING_POINT 1)
+set(USE_SSE 0)
```

Comment:
Will create a perf test and evaluate reasonable combinations of these parameters to measure CPU load.